nengo/utils/tests/test_numpy.py

```diff
@@ -21,3 +23,23 @@ def test_meshgrid_nd():
                   [[23, 42], [23, 42], [23, 42]]])]
     actual = meshgrid_nd(a, b, c)
     assert np.allclose(expected, actual)
+
+
+@pytest.mark.parametrize("shape", [(50,), (500, 2)])
```

Comment:
Just thinking but maybe the size of the first array should be 5 instead of 50. At some point, we may decrease the size of arrays that get copied (really, we should do an analysis and find at what point is generating the random numbers for larger array hashing more costly than copying). Anyway, having a smaller number would help ensure that even if we decrease the default, this still works. The other option is to set the `n` parameter on `array_hash` to explicitly set the point at which we randomly hash, to make sure that we test both cases.

Reply:
I've pushed an update that explicitly sets the `n` parameter in the tests. In either case though, I trust that if the default for `array_hash` is ever changed, that the unit tests will be looked at and changed accordingly as a part of that review.